docs/api/sentence.md

```diff
@@ -0,0 +1,14 @@
+# sentence
+
+This module contains libraries for sentence segmentation and sentence corpus construction
+
+## Module Overview
+
+::: remarx.sentence
+
+## Segmentation Functions
+
+::: remarx.sentence.segment
```

Comment:
The formatting for your example doesn't look right, do you want to fix as part of this PR?